products/BellHybrid/services/appmgr/ApplicationManager.cpp

```diff
@@ -45,6 +46,20 @@ namespace app::manager
         }
     }
 
+    auto ApplicationManager::handleDisplayLanguageChange(app::manager::DisplayLanguageChangeRequest *msg) -> bool
```

Comment:
It should probably be in the common part as this implementation is shared between Pure and Bell. 
The difference should be in the Pure implementation:
DBServiceAPI::InformLanguageChanged(this);

Reply:
So the idea behind this PR is such:
-`handleDisplayLanguageChange() ` is now virtual in ApplicationManagerCommon
ApplicationManagers for Pure and Bell have this method overriden:
- Pure uses the one from ApplicationManagerCommon without changes
- Bell has this method slightly modified with DBServiceAPI::InformLanguageChanged(this) removed. This was causing an assertion during language change and as i understand is not needed in Bell

Reply:
Applied changes suggested in the comment